pkgs/development/coq-modules/metacoq/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, mkCoqDerivation, which, equations, coq, version ? null }:
+with lib;
+
+mkCoqDerivation {
+  pname = "metacoq";
+  owner = "MetaCoq";
+
+  release."8.13".rev = "v1.0-beta2-8.13";
+  release."8.13".sha256 = "sha256-IC56/lEDaAylUbMCfG/3cqOBZniEQk8jmI053DBO5l8=";
+
+  inherit version;
+  defaultVersion = with versions; switch coq.version [
+    { case = range "8.11" "8.13"; out = "8.13"; }
```

Comment:
Is version 8.13 really compatible with Coq 8.11 and Coq 8.12?

Reply:
I'm not sure about this yet because I haven't gotten metacoq to build